invokeai/frontend/web/src/features/lora/store/loraSlice.ts

```diff
@@ -58,10 +60,18 @@ export const loraSlice = createSlice({
       }
       lora.weight = defaultLoRAConfig.weight;
     },
+    loraToggled: (state, action: PayloadAction<Pick<LoRA, 'id' | 'isEnabled'>>) => {
+      const { id, isEnabled } = action.payload;
+      const lora = state.loras[id];
+      if (!lora) {
+        return;
+      }
+      lora.isEnabled = isEnabled;
+    },
```

Comment:
"toggled" implies the new state will be the opposite of the previous state - so I'd expect "toggled" actions to not have an `isEnabled` boolean. `loraIsEnabledChanged` is clearer.

This is the establish project convention, used elsewhere, for ex:

```ts
    controlAdapterIsEnabledChanged: (state, action: PayloadAction<{ id: string; isEnabled: boolean }>) => {
    ...
    controlAdapterAutoConfigToggled: (state, action: PayloadAction<{ id: string }>) => {
```